views/ideas.html

```diff
@@ -17,7 +17,18 @@ <h2>{{ .Name }}</h2>
 {{ end }}
 {{ end }}
 
-    <h1 class=span>Ideas</h1>
+    <h1 class=span>Languages</h1>
+
+{{ range .Data.Languages }}
+    <a class="card color" href="//github.com/code-golf/code-golf/issues/{{ .ID }}" title="{{ .Title }}">
```

Comment:
We ought to be able to avoid duplicating the identical markup with block/define? https://pkg.go.dev/text/template#hdr-Actions

Reply:
Great! I've also never seen that templating system so I didn't bother trying to find out if/how it can be done. :)